src/avr/Kconfig

```diff
@@ -53,6 +55,7 @@ config MCU
     default "atmega32u4" if MACH_atmega32u4
     default "atmega1280" if MACH_atmega1280
     default "atmega2560" if MACH_atmega2560
+    default "atmega328p" if MACH_lgt8f328p
```

Comment:
Altering the internal name is going to cause a lot of confusion.  What's the requirement here - is it that the build must be done with `-mmcu=atmega328p` while the mcu needs to be declared to the host as "lgt8f328p"?